orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -9,31 +9,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# TODO move to preburn
 - name: Remove all old registries
   become: yes
   ignore_errors: yes
   ansible.builtin.file:
     path: /etc/apt/sources.list.d/*.list
     state: absent
-  when: preburn
-
-- name: Copy gpg key from codebase
-  copy:
-    src: jfrog.pub
-    dest: /tmp/jfrog.pub
```

Comment:
Can we delete the superfluous `jfrog.pub`?

Reply:
`/tmp` is cleaned-up with each VM restart.

Reply:
I mean the checked-in `magma/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub`.

Reply:
In this PR the file is still used in various places. But it will be removed in #14508.